code/modules/modular_computers/file_system/programs/security/camera.dm

```diff
@@ -36,7 +36,6 @@
 	size = 12
 	available_on_ntnet = TRUE
 	requires_ntnet = TRUE
-	required_access_download = ACCESS_HEADS
```

Comment:
That needs to stay.
We do not want to establish a culture similar to medical with suit-sensors where it becomes a expectation to constantly watch the cameras because you can.

The preferred option would be to add a actual "news" application that can access the same things the newscasters can access, as well as the journalist-feed (if one is present)
Or you could add a sub-type of the camera app that allows to only view the journalist/thunderdome network